asyncsleepiq/fuzion/core_climate.py

```diff
@@ -0,0 +1,33 @@
+"""Foundation for Core Climate for Fuzion SleepIQ API."""
+from __future__ import annotations
+
+from typing import Any
+
+from ..api import SleepIQAPI
+
+from ..core_climate import SleepIQCoreClimate
+
+from ..consts import SIDES_FULL, CoreTemps, Side
+
+
+class SleepIQFuzionCoreClimate(SleepIQCoreClimate):
+    """
+        CoreClimate (Also known as Heidi) representation for SleepIQ Fuzion API.
+        Heidi is the name of the climate calls in the SleepIQ API.
+        Controls heating and cooling.
+    """
+
+    async def set_mode(self, temperature: CoreTemps, time: int) -> None:
+        """Set core climate state through API."""
+        if time <= 0 or time > 360:
```

Comment:
Do the same as with the foot warmer (on main branch) and have the max time be a class variable

Reply:
Good call, updated it.